suite-native/module-settings/src/tests/utils.test.ts

```diff
@@ -0,0 +1,7 @@
+import { transformFiatCurrencyToSelectItem } from '../components/CurrencySelector';
```

Comment:
forget to move also test

Reply:
https://github.com/trezor/trezor-suite/pull/5895/commits/a54a081b813e4d5c0367770c18c3276d48792e06